dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathBuilder.java

```diff
@@ -75,8 +75,6 @@ private ClassPathBuilder() {}
       DependencyPath firstValue = Iterables.getFirst(multimap.get(key), null);
       pathToArtifact.put(key, firstValue.getLeaf());
     }
-    multimap.forEach((key, value) -> {});
```

Comment:
Why did this go away?

Reply:
This statement does not anything. (I think I forgot to remove while inspecting the content)